packages/amplify_datastore/lib/method_channel_datastore.dart

```diff
@@ -51,25 +55,24 @@ class AmplifyDataStoreMethodChannel extends AmplifyDataStore {
   /// This methods configure an event channel to carry datastore observe events. This
   /// can only be done after Amplify.configure() is called and before any observe()
   /// method is called.
-  Future<void> configure({String configuration}) async {
+  Future<void> configure({String? configuration}) async {
     // First step to configure datastore is to setup an event channel for observe
     return _channel.invokeMethod('setupObserve', {});
   }
 
   @override
   Future<List<T>> query<T extends Model>(ModelType<T> modelType,
-      {QueryPredicate where,
-      QueryPagination pagination,
-      List<QuerySortBy> sortBy}) async {
+      {QueryPredicate? where,
+      QueryPagination? pagination,
+      List<QuerySortBy>? sortBy}) async {
     try {
       final List<Map<dynamic, dynamic>> serializedResults =
-          await _channel.invokeListMethod('query', <String, dynamic>{
+          (await (_channel.invokeListMethod('query', <String, dynamic>{
         'modelName': modelType.modelName(),
         'queryPredicate': where?.serializeAsMap(),
         'queryPagination': pagination?.serializeAsMap(),
-        'querySort':
-            sortBy?.map((element) => element?.serializeAsMap())?.toList()
-      });
+        'querySort': sortBy?.map((element) => element.serializeAsMap()).toList()
+      })))!;

```

Comment:
nit: indentation for readability

Reply:
Thanks!  Updated indentation for that file